### PR TITLE
:shirt: Small fixes and improvements

### PR DESCRIPTION
**Pull request by o-laurent**
- Improve numpy usage
- Improve the MC-Dropout wrapper & tutorial

### CODE REVIEW

File: auto_tutorials_source/tutorial_mc_dropout.py [hunk 1]
```diff
@@ -63,18 +61,18 @@
...
 # %%
 # 3. The Loss and the Training Routine
 # ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-# This is a classification problem, and we use CrossEntropyLoss as the likelihood.
+# This is a classification problem, and we use CrossEntropyLoss as the (negative-log-)likelihood.
 # We define the training routine using the classification training routine from
-# torch_uncertainty.routines.classification. We provide the number of classes
-# and channels, the optimizer wrapper, and the dropout rate.
+# torch_uncertainty.routines. We provide the number of classes
+# the optimization recipe, and tell the routine that our model is an ensemble at evalutation time.
```
Comment (alafage) [inline auto_tutorials_source/tutorial_mc_dropout.py:75]: evaluation

File: torch_uncertainty/datasets/regression/uci_regression.py [hunk 2]
```diff
@@ -225,9 +227,10 @@ def download(self) -> None:
 
     def _make_dataset(self) -> None:
         """Create dataset from extracted files."""
-        if not util.find_spec("pandas"):
+        if not pandas_installed:
```
Comment (alafage) [inline torch_uncertainty/datasets/regression/uci_regression.py:230]: ``pandas=False`` is not initialized

File: torch_uncertainty/models/wrappers/mc_dropout.py [hunk 3]
```diff
@@ -1,6 +1,8 @@
 import torch
 from torch import Tensor, nn
 
+DROPOUT_MODULES = (nn.Dropout, nn.Dropout1d, nn.Dropout2d, nn.Dropout3d)
```
Comment (alafage) [inline torch_uncertainty/models/wrappers/mc_dropout.py:4]: use ``nn._DropoutNd``

**Review by alafage — APPROVED** [review]
